### PR TITLE
Add note about Xcode on El Captain

### PR DESCRIPTION
**Pull request by hultberg**
I have just built atom on my mac for the first time. I took me about 2 hours to do it successfully, because installing XCode with App Store **then** opening it was not so obvious. This PR adds a note about it before the little code snippet.

### CODE REVIEW

File: docs/build-instructions/os-x.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 
 ## Instructions
 
+ **El Captain**: Install XCode from App Store and then open it before running build script.
```
Comment (albinekcom) [inline docs/build-instructions/os-x.md]: It should be `Xcode`, not `XCode`.

**Comment by lee-dohm** [comment]
Did you have the Command Line Tools for XCode already installed as outlined in the pre-requisites? You shouldn't need to install all of XCode if you install the command line tools. (Though you can also get the command line tools by installing XCode.)

**Comment by hultberg** [comment]
@lee-dohm In fact I did. When I ran `xcodebuild --version` I got an error regarding Xcode not being installed, but was available in Command Line. As I kinda... fixed this problem I'm currently unable to retrieve the error message I received at that point. I'll do some duckducking to see if I stumble upon it.

**Comment by hultberg** [comment]
@lee-dohm I uninstalled Xcode as downloaded from App Store and ran a new build. While "Installing apm" and "Installing build modules" I get the following errors:

````
xcode-select: error: tool 'xcodebuild' requires Xcode, but active developer directory '/Library/Developer/CommandLineTools' is a command line tools instance
````

Also, when running the command as outlined in the pre-requisites, I get:
````
xcode-select: error: command line tools are already installed, use "Software Update" to install updates
````

**Comment by sadick254** [comment]
Hey @hultberg I know this PR has been open for a long time and we didn't respond to it early enough. 
Information on `Xcode` has been included on the [atom flight manual](https://flight-manual.atom.io/hacking-atom/sections/hacking-on-atom-core/#requirements). We will be closing this PR for now. We can reopen this if you find the documentation unclear. 🙇

**Pull request closed without merging**